projects/PeerTube/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+
+{
+  metadata = {
+    summary = "A decentralised streaming video platform";
+    subgrants = [
+      "PeerTube"
+      "PeerTubeSearch"
+      "Peertube-Transcode"
+      "Peertube-Livechat"
+      "PeerTubeDesktop"
+      "PeerTube-mobile"
+    ];
+    links = {
+      website = {
+        text = "Website";
+        url = "https://joinpeertube.org/";
+      };
+      docs = {
+        text = "Documentation";
+        url = "https://docs.joinpeertube.org/";
+      };
+    };
+  };
+
+  nixos.modules.services = {
+    peertube = {
+      name = "peertube";
+      module = ./services/peertube/module.nix;
+      examples.basic = {
+        module = ./services/peertube/examples/basic.nix;
+        description = "Basic configuration mainly used for testing purposes.";
+        tests.basic = import ./services/peertube/tests/basic.nix args;
```

Let's call this `peertube-plugins` as it was originally, since that's what the test is doing

```suggestion
        tests.peertube-plugins = import ./services/peertube/tests/plugins.nix args;
```